### PR TITLE
My code review

### PR DESCRIPTION
**Pull request by den-cloud**
In this  project I was reviewing my project from the trails and add necessary, to have my project pass all no linter errors requirement.
 
I used html and css to build this project.

### CODE REVIEW

**Review by den-cloud — COMMENTED** [review]
I added new html and css code.
In the project , I added folder called gitgub/workflows 
added files linters.yml, gitignore, hintrc, stylelintrc and a read.md

run npx hint to find and correct errors
run npx stylelint to and correct erros

File: index.html [hunk 1]
```diff
@@ -5,30 +5,28 @@
...
     <header id="header">
         <!--headers-->
         <h1>Samsung Televisions</h1>
-
-        <img src="IMAGES/LOGO.jfif" alt="Samsung Logo" id="header-img">
+        <img src="./IMAGES/LOGO.jfif" alt="Samsung Logo" id="header-img">
         <nav id="nav-bar">
             <a class="nav-link" href="#home">Home</a>
             <a class="nav-link" href="#service">Service</a>
             <a class="nav-link" href="#product">Product</a>
```
Comment (Dipeshtwis) [inline index.html]: This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @den-cloud, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] Let's use the GitHub inline comment. See the example inline comment review below.
- [x] wrote at least 2 comments in the review
- [x] All comments should be polite, ask for "why" the changes requested, and make a clear comment. See this article on [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for more details.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary like this current review. :wink:

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by den-cloud — COMMENTED** [review]
I added new html and css code.
In the project , I added folder called gitgub/workflows
added files linters.yml, gitignore, hintrc, stylelintrc and a read.md

run npx hint to find and correct errors
run npx stylelint to and correct erros

File: css/style.css [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+  font-family: "Roboto", sans-serif;
+}
+
 #header {
-  top: 0px;
   width: 100%;
   height: 100%;
   display: flex;
   justify-content: space-between;
-  align-items: center;
```
Comment (den-cloud) [inline css/style.css:7]: align-item was giving improper outcome, removed it

File: css/style.css [hunk 3]
```diff
@@ -1,16 +1,30 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
 #header {
-  top: 0px;
```
Comment (den-cloud) [inline css/style.css:2]: align-item was giving improper outcome, removed it

**Review by den-cloud — COMMENTED** [review]
I added new html and css code.
In the project , I added folder called gitgub/workflows
added files linters.yml, gitignore, hintrc, stylelintrc and a read.md

run npx hint to find and correct errors
run npx stylelint to and correct erros

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @den-cloud 👋🏾 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Just to clarify, the focus on this milestone is based on "how" well you communicate and perform a code review, by being friendly, clear, polite, professional, and using the GitHub code review features appropriately. The project should have a few errors for you to perform a review. Therefore, for this project to be approved you need to work on ALL the changes, the previous reviewer asked of you. Kindly read this article on [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**